app/src/main/assets/apache_licence_notice.html

```diff
@@ -229,6 +229,8 @@
     Apache Taverna Mobile
 Copyright 2015-2018 The Apache Software Foundation
 
+1.Disclaimer: myExperiment's Logo is nothing to do with Apache and is copyright to myExperiment.
```

Comment:
move it line no 236. just add `myExperiment's Logo is nothing to do with Apache and is copyright to myExperiment.`.

Reply:
I have done the changes.